com.synology.synology-note-station.metainfo.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.synology.synology-note-station</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://www.synology.com/en-global/company/legal/terms_EULA</project_license>
```

Comment:
I think I asked it before but can't find it... Is redistribution allowed under the license?

Reply:
I am not a lawyer.  
If its not allowed the app  https://flathub.org/apps/com.synology.SynologyDrive  violates it too. 
Here a part of the "meta.xml"
"NOTE: This wrapper is not verified by, affiliated with, or supported by Synology Inc." 
I have tried to contact them. Without success yet.

Reply:
They use extra-data to avoid that, which you are not doing https://github.com/flathub/com.synology.SynologyDrive/blob/da5154e9936c0d11c73c14eaac5d609dd146fcc3/com.synology.SynologyDrive.yaml#L47

Reply:
https://bbhtt.github.io/flatpak-docs/module-sources.html#extra-data

Reply:
than i will implement that.

Reply:
I have implemented that now. 
copied most from https://github.com/flathub/com.synology.SynologyDrive/blob/da5154e9936c0d11c73c14eaac5d609dd146fcc3/com.synology.SynologyDrive.yaml#L47